src/Quadrature/quadrature.jl

```diff
@@ -154,14 +172,14 @@ end
 ######################
 
 """
-    FacetQuadratureRule{shape}([::Type{T}, [quad_rule_type::Symbol,] order::Int)
+    FacetQuadratureRule{shape}([::Type{T}, [quad_rule_type::Symbol|quad_rule_type::NTuple{Symbol},] order::Int)
     FacetQuadratureRule{shape}(face_rules::NTuple{<:Any, <:QuadratureRule{shape}})
     FacetQuadratureRule{shape}(face_rules::AbstractVector{<:QuadratureRule{shape}})
 
 Create a `FacetQuadratureRule` used for integration of the faces of the refshape `shape` (of
 type [`AbstractRefShape`](@ref)). `order` is the order of the quadrature rule.
-`quad_rule_type` is an optional argument determining the type of quadrature rule.
-Currently the :polyquad, `:legendre` and `:lobatto` rules are implemented, depending on the reference element.
+It defaults to the default rules for the individual face reference elements. Elements with mixed facet types
+requires passing a tuple of symbols for the rule, one for each facet type.
```

Comment:
The ordering of the `quad_rule_type` is not clear/obvious here. I'm not sure if we want to introduce this as public API now? 

Would it be better (at least for now), to rather say that if you want non-default rules, you need to pass the `face_rules` manually?

Reply:
Indeed. I also struggle a bit with the wording in the docstring. It is essentially implemented as "first triangle, then quad" in the sense of "increasing size".

Reply:
bump